src/config/mongoose_config_spec.erl

```diff
@@ -172,6 +172,9 @@ general() ->
                  <<"sm_backend">> => #option{type = atom,
                                              validate = {module, ejabberd_sm},
                                              wrap = global_config},
+                 <<"component_backend">> => #option{type = atom,
+                                                    validate = {module, mongoose_component},
```

Comment:
I'm not quite sure I understand how this validation works. Could you explain how we are making sure that `cets` and `mnesia` are the only valid options for this?

Reply:
It accepts any existing module with the name `mongoose_component_*`. The code is correct.

Reply:
Thanks